float64[7] ik_pos
---
float64[6] sol1
float64[6] sol2
float64[6] sol3
float64[6] sol4
float64[6] sol5
float64[6] sol6
float64[6] sol7
float64[6] sol8
